### ISSUE TITLE
Evaluate a new deployment plan and scalability proposal for code

### ISSUE THREAD

**Issue by rvizarreta** [rvizarreta/Mx2_SlowControlsDisplay#15]
- In the future we need to consider adding more modules with different channels. We will handle this once we evaluate how to re-write a modular code.
- Review https://linuxhint.com/tmux_vs_screen/
- One good idea for naming convention will be having one InfluxDB database per Module. There we don't need to change the channels names for each of the 4 modules. Module 0 channel u1, module 1 channel u1, etc.
- Consider that some units might not be able to turn on remotely.

**Comment by rvizarreta** [comment]
I already started writing a modular code based on the 'don't repeat yourself' principle. I first made a class PSU that contains all the methods required for this, but then I realized this will work only for the mpods because (i.e.) gizmo has a different way to collect data, so those methods will be useless. I will make a main PSU class and then each particular device will have its own class to extend from this. I am not sure how this will change the json file that currently works as a metadata DB, but will probably stay the same. Once the gizmo implementation is done I can start thinking in the proposal for FastAPI.
